cli/cmd/deploy.go

```diff
@@ -53,21 +55,47 @@ func deploy(force bool, ignoreCache bool) {
 		errors.Exit(err)
 	}
 
-	zipInput := &zip.Input{
+	params := map[string]string{
+		"force":       s.Bool(force),
+		"ignoreCache": s.Bool(ignoreCache),
+	}
+
+	configBytes, err := ioutil.ReadFile("cortex.yaml")
+	if err != nil {
+		errors.Exit(errors.Wrap(err, "failed to read configuration file"))
```

Comment:
We have an error for this: `ErrorReadConfig`. In fact the error messages are very similar, great minds think alike :) feel free to change "config" to "configuration" if you prefer